uptime_service_validation/database/create_tables.sql

```diff
@@ -35,12 +35,13 @@ CREATE TABLE IF NOT EXISTS nodes (
 	block_producer_key TEXT,
 	updated_at TIMESTAMPTZ(6),
 	score INT,
-	score_percent NUMERIC(10,2),
+	score_percent NUMERIC(5,2),
```

Comment:
Is there a specific reason to reduce accuracy here or is it just that we don;t use that much accuracy anyway?

Reply:
this field holds percentage so <0.00 - 100.00>, NUMERIC(5,2) allows for max 999.99, but anything above 100.00 is basically wrong, but that's good enough I think.